scripts/generate_markdown_files.py

```diff
@@ -1,40 +1,18 @@
-import os
-import importlib.util
 from inspect import getmembers, isclass
-import yaml
+from ruamel.yaml import YAML
```

Comment:
you need to install this package as part of the pipeline.

Reply:
The pipeline has not been created yet.